operations/restrictions.md

```diff
@@ -144,6 +146,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `Id` | string | required | Unique identifier of the restriction. |
 | `ServiceId` | string | required | Unique identifier of the [Service](services.md#service). |
 | `ExternalIdentifier` | string | optional | External identifier of the restriction. |
+| `Origin` | string | required | [Restriction origin](#restriction-origin) |
```

Comment:
If this is required in the response, then we need to add "Origin" to the JSON in the response example.

Reply:
Done